preprocessy/scaling/errors.py

```diff
@@ -5,3 +5,6 @@ class Error(Exception):
 class ArgumentsError(Error):
     def __init__(self, message):
         self.message = message
+
+    def __repr__(self):
+        return f"ArgumentsError(message={self.message})"
```

Comment:
I don't think a ```__repr__``` function is required for this class.

Reply:
Will get it removed